### PR TITLE
Add missing types

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: src/Utils/Reflection.php [hunk 1]
```diff
@@ -188,7 +188,7 @@ public static function expandClassName(string $name, \ReflectionClass $context):
 	}
 
 
-	/** @return array of [alias => class] */
+	/** @return array<string, string> of [alias => class] */
```
[the remark this answers is not in the document]
Comment (mabar) [reply src/Utils/Reflection.php:191]: PHPStan supports `interface-string` and `trait-string` as an alias to `class-string`. `class-string` should be fine, classes, interfaces and traits are all treated the same way in this context
https://phpstan.org/r/d8b857b5-b92b-46e4-aeed-f0545d8d536d

[the remark this answers is not in the document]
Comment (enumag) [reply src/Utils/Reflection.php:191]: I see... didn't know that

[the remark this answers is not in the document]
Comment (enumag) [reply src/Utils/Reflection.php:191]:
```suggestion
	/** @return array<string, class-string> of [alias => class] */
```

**Comment by dg** [comment]
thanks

**Pull request merged**